daemon/queue/NzbFile.cpp

```diff
@@ -391,15 +391,32 @@ void NzbFile::ProcessFiles()
 
 	if (m_password)
 	{
-		ReadPassword();
+		ReadPasswordFromNzb();
+	}else{
```

Comment:
Please, move 'else' and '{' to the next and separate lines to be consistent with the code style.